packages/mds-agency/tests/db-integration-tests.ts

```diff
@@ -43,6 +43,7 @@ import {
 import db from '@mds-core/mds-db'
 import cache from '@mds-core/mds-cache'
 import stream from '@mds-core/mds-stream'
+import { shutdown as socketShutdown } from '@mds-core/mds-web-sockets'
```

Comment:
Would prefer `import socket from '@mds-core/mds-web-sockets'` to be consistent with db/cache/stream @avatarneil